Makefile

```diff
@@ -1,5 +1,5 @@
-OK_COLOR=\033[32;01m
-NO_COLOR=\033[0m
+OK_COLOR := $(shell tput -Txterm setaf 2)
```

Comment:
Does `-Txterm` is a necessary parameter? What happens if I'm using non-xterm terminal?

Reply:
@Unviray please check this comment ^^